Paccy10/activo-api
# [feature] send registration email

**What does this PR do?**

- send registration email

**Description of Task to be completed?**

- Add `django_rq` to run background tasks
- Configure app to send an email on registration 

**How should this be manually tested?**

- Send a `POST` request to `/users/`
- Check your inbox, you will see an email with email and OTP

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A
